config/schema/format_strawberryfield.schema.yml

```diff
@@ -138,6 +144,16 @@ field.formatter.settings.strawberry_metadata_formatter:
       type: string
     metadatadisplayentity_uselabel:
       type: string
+    mediasource:
```

Comment:
@marlo-longley this does not make sense. Why are you adding Mirador settings to strawberry_metadata_formatter?

Reply:
<img width="1263" alt="Screen Shot 2020-02-19 at 11 21 59 PM" src="https://user-images.githubusercontent.com/1328900/74910159-b6680600-536e-11ea-8126-2e42bb00b7bb.png">
That was why?

Reply:
Can you **please** point me to the code that writes that? I see none of these settings being written by us... https://github.com/esmero/format_strawberryfield/blob/8.x-1.0-beta2/src/Plugin/Field/FieldFormatter/StrawberryMetadataTwigFormatter.php#L153-L193
I'm still wondering if need help understanding the life cycle of a config. But code, again, is what rules. That is also why i asked you to save, one by one each display mode to see if any of our schemas where generating the right settings on save.